### DISCUSSION TITLE
Image rescale doesn't work in v0.4.2

### DISCUSSION THREAD

**Discussion by hanhanwu** [Kanaries/pygwalker#404] [Q&A] [answered]
I checked this solution https://github.com/Kanaries/pygwalker/discussions/293, but it doesn't work for me, no matter changing to "fixed" or "container", my image size doesn't change... I also tried adjust the width and height, but the only change the container size not the image size.

Does this happen to you too?

**Comment by longxiaofei** [comment, marked as answer]
I currently cannot reproduce this problem in 0.4.2.

Can you provide more details? For example, chart type, operation steps?

https://github.com/Kanaries/pygwalker/assets/28337703/8fb8a0e3-c385-43e6-a044-9114c38f7648

**Reply by hanhanwu** [reply]
I just realized my Chrome can't resize but it worked on another browser. Maybe my Chrome blocked JS. Thank you!
